operators/advanced_network/managers/dpdk/adv_network_dpdk_mgr.cpp

```diff
@@ -49,9 +50,12 @@ struct RxWorkerParams {
   int num_segs;
   uint32_t batch_size;
   struct rte_ring* ring;
+  struct rte_ring* lb_ring;          // Ring used between TX and RX when in SW loopback mode
```

Comment:
```suggestion
  struct rte_ring* sw_lb_ring;          // Ring used between TX and RX when in SW loopback mode
```

Reply:
This may be used for other types of loopback, so leaving the name for now.